.onepipeline.yaml

```diff
@@ -0,0 +1,125 @@
+    set +e
+    oc new-project fybrik-system
+    set -e
+    kubectl get po -o wide
+test:
+  abort_on_failure: false
+  image: wcp-ibm-streams-docker-local.artifactory.swg-devops.com/pipelines-tutorial/compliance-baseimage-with-tkn:2.26.1
+  script: |
+    #!/usr/bin/env bash
+    #cd ../"$(load_repo app-repo path)"
+    if [[ "$PIPELINE_DEBUG" == 1 ]]; then
+      trap env EXIT
+      env
+      set -x
+    fi
+    set -x
+    env
+    ls $WORKSPACE
+    oc login $(get_env openshift_cluster) -u $(get_env admin_user) -p $(get_env admin_pw) --insecure-skip-tls-verify=true
+    set +e
+    oc new-project fybrik-system
+    oc project fybrik-system
+    oc delete pipelinerun --all --wait
+    set -e
+    export GH_TOKEN=$(get_env different-git-token)
+    export git_user=$(get_env git-user)
+    . pipeline/source-internal.sh
+    github_workspace=`pwd`/../mesh-for-data run_tkn=$(get_env run_tkn) skip_tests=false bash -x pipeline/bootstrap-pipeline.sh fybrik-system
+    for i in $(oc get po --no-headers | grep run-unit | cut -d' ' -f1); do oc logs $i -c step-unit-test > $i.logs ; save_artifact test log_$i=$i.logs; done
+containerize:
+  dind: true
+  image: wcp-ibm-streams-docker-local.artifactory.swg-devops.com/pipelines-tutorial/compliance-baseimage-with-tkn:2.26.1
+  script: |
+    #!/bin/bash
+    set -x
+    if [[ "$PIPELINE_DEBUG" == 1 ]]; then
+      trap env EXIT
+      env
+      set -x
+    fi
+    env
+    ls $WORKSPACE
+    oc login $(get_env openshift_cluster) -u $(get_env admin_user) -p $(get_env admin_pw) --insecure-skip-tls-verify=true
+    oc project fybrik-system
+    . pipeline/common_functions.sh
+    cat > ${TMP}/streams_csv_check_script.sh <<EOH
+    #!/bin/bash
+    set -x
+    oc get pipelinerun --no-headers
+    oc get pipelinerun --no-headers | grep -e Failed -e Completed
+    EOH
+    chmod u+x ${TMP}/streams_csv_check_script.sh
+    try_command "${TMP}/streams_csv_check_script.sh"  40 true 5
+    pipelinerun=$(oc get pipelinerun --no-headers | cut -d' ' -f1)
+    oc get pipelinerun $pipelinerun -o yaml > /tmp/details
+    #cat /tmp/details
+    #yq r /tmp/details 'status.taskRuns[*].status.taskResults'
+    #yq r /tmp/details 'status.taskRuns[*].status.taskResults' | grep FULL_IMAGE -A1
+    #yq r /tmp/details 'status.taskRuns[*].status.taskResults' | grep FULL_IMAGE -A1 | grep value:
+    #yq r /tmp/details 'status.taskRuns[*].status.taskResults' | grep FULL_IMAGE -A1 | grep value: | cut -d':' -f2-
+  
+    # Create the namespace if needed to ensure the push will be can be successfull
+    ICR_REGISTRY_NAMESPACE=$(get_env registry-namespace)
+    echo "Checking registry namespace: ${ICR_REGISTRY_NAMESPACE}"
+    IBM_LOGIN_REGISTRY_REGION=$(cat /config/registry-region | awk -F: '{print $3}')
+    ibmcloud login --apikey @/config/api-key -r "$IBM_LOGIN_REGISTRY_REGION"
+    NS=$( ibmcloud cr namespaces | sed 's/ *$//' | grep -x "${ICR_REGISTRY_NAMESPACE}" ||: )
```

Comment:
ibmcloud is not a prerequisite for fybrik.
Also, I believe IBM specifics should not be used in open-source repo. @ronenkat ?

Reply:
The one-pipeline automation will not run, by default, on the open source code.  But as I noted before, if we want to remove any reference to it in this repo, we can.